editor/edit-post/modes/visual-editor/style.scss

```diff
@@ -39,6 +39,14 @@
 	}
 }
 
+// This is a focus style shown for blocks that need an indicator even when in an isEditing state
+// like for example an image block that receives arrowkey focus.
+.editor-visual-editor .editor-block-list__block:not( .is-selected ) .editor-block-list__block-edit:focus  {
```

Comment:
Shouldn't this be moved to the `editor/components/block-list/style.scss` file? Thinking this style could be usefulll regardless of the layout.